index.css

```diff
@@ -78,8 +88,372 @@
+  }
+
+  .list-x .third-list-x {
+    margin-left: 20%;
+  }
+
+  .print-data button:hover {
+    background-color: white;
+    color: black;
+    border: solid;
+  }
+
+  .about-me {
+    background-color: rgba(28, 26, 25, 1);
+    margin-top: 33%;
+  }
+
+  .background-holder {
+    display: flex;
+    flex-direction: column;
+    background-image: url("./logos/About Me.png");
+    background-repeat: no-repeat;
+    background-size: 300px 500px;
+    background-position: right bottom;
+  }
+
+  .about-me h2 {
+    color: rgba(255, 107, 0, 1);
+    font-size: 800%;
+    font-family: 'Crete Round', serif;
+    text-align: center;
+    padding-top: 35%;
+    margin-left: 8%;
+    margin-right: 8%;
+  }
+
+  .about-me p {
+    font-family: 'Inter', sans-serif;
+    font-size: 200%;
+    margin-left: 8%;
+    margin-right: 8%;
+    margin-top: -10%;
+    margin-bottom: 8%;
+  }
+
+  .about-me button:hover {
+    background-color: white;
+    color: black;
+    border: solid;
+    cursor: pointer;
+  }
+
+  .about-me hr {
+    margin-left: 8%;
+    margin-right: 8%;
+    margin-bottom: 30%;
+  }
+
+  .lists {
+    padding-bottom: 20%;
+  }
+
+  .lists li {
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    list-style: none;
+    text-decoration: none;
+    margin-bottom: 6%;
+  }
+
+  .lists h3 {
+    font-family: 'Inter', sans-serif;
+    font-size: 300%;
+    text-align: center;
+  }
+
+  .attach-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .interesting {
+    margin-left: 8%;
+    margin-right: 8%;
+  }
+
+  .interesting h4 {
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+    margin-top: 20%;
+    font-size: 350%;
+    margin-bottom: 20%;
+  }
+
+  .interesting input,
+  textarea {
+    font-family: 'Inter', sans-serif;
+  }
+
+  .interesting input {
+    font-size: 150%;
+    border: none;
+    margin-bottom: 2%;
+    width: 100%;
+  }
+
+  .interesting .end-name {
+    margin-bottom: 10%;
+    border-color: rgba(219, 216, 215, 0.904);
+  }
+
+  .interesting .end-address {
+    margin-bottom: 7%;
+    border-color: rgba(219, 216, 215, 0.904);
+  }
+
+  .interesting .text-area {
+    font-size: 150%;
+    font-style: italic;
+    background-color: #fbf8f7;
+    width: 100%;
+    border: none;
+    margin-bottom: 5%;
+  }
+
+  .interesting button:hover {
+    background-color: white;
+    color: black;
+    border-color: black;
+  }
+
+  .interesting .end-page {
+    border-color: rgba(219, 216, 215, 0.904);
+    margin-bottom: 7%;
+    width: 100%;
+  }
+
+  .footer ul {
+    display: flex;
+    list-style-type: none;
+    align-items: center;
+    justify-content: center;
+    margin-right: 12%;
+    margin-bottom: 10%;
+  }
+
+  .footer li {
+    margin-right: 0%;
+    color: #505f79;
+    margin-left: 8%;
+    padding-left: -4%;
+  }
+
+  .footer a img {
+    width: 30px;
+    height: 30px;
+  }
+
+  .footer hr {
+    margin-left: 30%;
+    margin-right: 30%;
```

Comment:
- [ ] Kindly use pixels instead percentages for the font size and margins to be similar to the Figma design.